io/net/sriov_device_test.py

```diff
@@ -158,6 +158,8 @@ def test_add_logical_device(self):
             self.device_add_remove(slot, port, mac, '', 'add')
             if not self.list_device(mac):
                 self.fail("failed to list logical device after add operation")
+            if not self.find_device(mac):
+                self.fail("MAC address differs in linux")
```

Comment:
why two  time calling the function it can be performance overhead, can this be used like 

```
device = self.find_device(mac)
if not device:
  self.fail("MAC address differs in linux")
  
  ```

Reply:
Done